src/System.CommandLine.DragonFruit/CommandLine.cs

```diff
@@ -116,6 +116,18 @@ public static class CommandLine
                 command.AddOption(option);
             }
 
+            if (method.GetParameters()
+                      .FirstOrDefault(p => p.Name == "arguments" || 
+                                           p.Name == "args" ||
+                                           p.Name == "argument") is ParameterInfo argsParam)
```

Comment:
Can we change the method signature to take in an optional collection of names to match on for arguments, and fall back to using this set if that collection is not specified. That would also give (albeit non-obvious) method for disabling/configuring this behavior.

Reply:
This functionality is only available in DragonFruit which is so far not configurable, so I'd prefer to wait on making this more flexible until we decide whether that's something to pursue.